packages/react-client/src/ReactFlightClient.js

```diff
@@ -446,7 +457,7 @@ function createModelResolver<T>(
       value: null,
     };
   }
-  return value => {
+  return (value: any) => {
```

Comment:
Would it make more sense to add a FlowFixme[missing-local-annot] like you did in other cases?

Reply:
Yeah, I think flow auto-inferred some anys (probably because they're called with an any context somewhere), but the flow fix me makes more sense.